.github/workflows/build_docs.yaml

```diff
@@ -82,7 +83,7 @@ jobs:
   upload:
     runs-on: ubuntu-latest
     needs: build_docs
-    if: github.repository == 'pytorch/torchtune' && github.event_name == 'push' && (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v'))
+    if: github.repository_owner == 'pytorch' && github.event_name == 'push' && (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v'))
```

Comment:
I'm curious about this one in particular.. why isn't `github.repository == 'pytorch/torchtune'` sufficient? My assumption is that forks would have a separate repository name, is that not the case?

Reply:
it totally was sufficient, i just changed it for consistency so someone looking can see that it’s the same across the workflows